backend/src/main/java/org/eclipse/tractusx/semantics/registry/repository/ShellRepository.java

```diff
@@ -54,15 +55,13 @@ public interface ShellRepository extends PagingAndSortingRepository<Shell, UUID>
     @Query(
             "select s.id_external from shell s where s.id in (" +
                     "select si.fk_shell_id from shell_identifier si " +
-                    "join (values :keyValueCombinations ) as t (input_key,input_value) " +
-                    "ON si.namespace = input_key " +
-                        "AND si.identifier = input_value " +
-                        "AND (si.external_subject_id is null or :tenantId = :owningTenantId or si.external_subject_id = :tenantId) " +
+                    "where concat(si.namespace,si.identifier) in (:keyValueCombinations) " +
```

Comment:
This we need to change once https://github.com/spring-projects/spring-data-relational/pull/1356 is released, correct?

Reply:
correct.